.travis/after_success.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Check if it is a pull request
+if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
+    echo -e "Pull Request, not pushing a build"
+    exit 0;
+fi
+
+# If current dev branch is master, push to build repo master
+if [ "${TRAVIS_BRANCH}" = "master" ]; then
+    .travis/release.sh
+fi
+
+# If current branch dev branch is stable/foo, push to build repo stable
```

Comment:
@Kinlaw can you update this too in your next commit? Should say "# If current dev branch is stable/foo, push to build repo stable`

Remove the first occurrence of "branch"